hedy.py

```diff
@@ -2127,18 +2142,6 @@ def is_program_valid(program_root, input_string, level, lang):
             raise exceptions.InvalidSpaceException(level=level, line_number=line, fixed_code=fixed_code, fixed_result=result)
         elif invalid_info.error_type == 'print without quotes':
             # grammar rule is agnostic of line number so we can't easily return that here
-            program_repair.make_mutants(input_string, line)
```

Comment:
so here we would do something like `if repair_enabled: mutation_repair(input_string, line, level, lang)`